src/compile/mark/init.ts

```diff
@@ -30,9 +30,23 @@ export function normalizeMarkDef(mark: Mark | MarkDef, encoding: Encoding<string
   if (specifiedFilled === undefined) {
     markDef.filled = filled(markDef, config);
   }
+
+  // set cursor, which should be pointer if href channel is present unless otherwise specified
+  const specifiedCursor = markDef.cursor;
+  if (specifiedCursor === undefined) {
+    markDef.cursor = cursor(markDef, encoding);
+  }
+
   return markDef;
 }
 
+function cursor(markDef: MarkDef, encoding: Encoding<String>) {
+  if (encoding.href || markDef.href) {
```

Comment:
Do we have href as a mark config?  If so, should we consider those too?

Reply:
I didn't think so but I will check.